ext/project_data.yaml

```diff
@@ -13,14 +13,6 @@ gem_require_path: 'lib'
 gem_test_files: 'spec/**/*'
 gem_executables: 'hiera'
 gem_default_executables: 'hiera'
-gem_platform_dependencies:
-  x86-mingw32:
-    gem_runtime_dependencies:
-      win32console: '1.3.2'
-      win32-dir: '~> 0.4.8'
-  x64-mingw32:
-    gem_runtime_dependencies:
-      win32-dir: '~> 0.4.8'
 bundle_platforms:
   x86-mingw32: mingw
   x64-mingw32: x64_mingw
```

Comment:
The `Gemfile` does lookup the `bundle_platforms` but it only uses it if `gem_platform_dependencies` is also defined. The only reason for looking up `bundle_platforms`, is because we do it once outside the loop. It could be moved inside the loop to make the dependency more explicit.

That said, I'm fine leaving it as-is.